site/en/r2/guide/migration_guide.ipynb

```diff
@@ -777,8 +777,8 @@
         "\n",
         "  return image, label\n",
         "\n",
-        "train_data = mnist_train.map(scale).shuffle(BUFFER_SIZE).batch(BATCH_SIZE).take(5)\n",
-        "test_data = mnist_test.map(scale).batch(BATCH_SIZE).take(5)"
+        "train_data = mnist_train.map(scale).shuffle(BUFFER_SIZE).batch(BATCH_SIZE)\n",
```

Comment:
From the text, I think 'take(5)` should stay, but move these lines to the following code cell. @yashk2810 to confirm

Reply:
Yup, the .take(5) should stay!